sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -737,4 +739,36 @@ export class JsonRpcProvider extends Provider {
       );
     }
   }
+
+  async getDelegatedStake(address: string): Promise<DelegatedStake[]> {
+    try {
+      const resp = await this.client.requestWithType(
```

Comment:
Could you add a check for the address? Example: https://github.com/MystenLabs/sui/blob/a43e62a3ca6fc46a980faee5967e6f7628a2910a/sdk/typescript/src/providers/json-rpc-provider.ts#L295